### DISCUSSION TITLE
mine pit infilling with tailings/rock

### DISCUSSION THREAD

**Discussion by RyanConway91** [MODFLOW-ORG/modflow6#1735] [Q&A]
I want to simulate mine pit infilling after mining completion. The mine pit will fill up with groundwater eventually, but will also be part of the process water system at the mine. For this reason, I simulated the mine pit during infilling using LAK so I can simulate the impacts of other parts of the water budget on the pit. 

Now the mine is planning to also infill the pit lake with rock and/or tailings. Is appropriate to still use LAK for this?

The LAK covers many layers vertically, so I could use TVK to simulate the deposition in the LAK. I have never thought much about the k of the materials inside the LAK. Are cells inside a multilayer layer LAK "active" or they they removed from the model solution and replaced with the LAK cells? 

Another thought I had was to try and modify the stage-volume-area curves to account the infilling (the added material will use up volume). However this would be time-varying as more material is added to the LAK through time and I don't think time-varying stage-volume is possible in LAK. 

Or I could just not use LAK and use high k cells where the mine pit is open and use tvk to fill in the tailings/rock as it is deposited. However then I will not be able to account for the other pieces of the water balance easily (ET, precipitation, engineered pumping/discharge)

Any advice?

**Comment by aprovost-usgs** [comment]
Hi Ryan,

My hat's off to you for the creative approaches you've considered.  But I agree they each appear to have their limitations: the lake is its own control volume that doesn't interact with TVK; there's no built-in mechanism for changing the stage-volume relation dynamically (apart from resorting to the MODFLOW API); and using high-k cells would mean giving up LAK's control over ET, etc.

If you knew that the stage in your pit would never go below the maximum (final) elevation of your infill, maybe you could use a hybrid approach – a lake on top of initially high-k cells whose k is lowered over time using TVK?  I'm guessing that assumption is too restrictive, though.  It's more than just saying the stage is always above the infill – it's saying that the stage is always above the highest the infill will ever get.

You could also consider doing it as a series of simulations, with infill being added and the model being updated between simulations.  If you wanted to account for the storage change in the lake due to adding infill, you'd need to calculate that and adjust the lake stage accordingly (in addition to the lake bottom) between simulations.

I hope this is helpful.

Best,

-- Alden

Alden Provost
US Geological Survey
Integrated Modeling and Prediction Division
703-648-5883 (office)


________________________________
From: RyanConway91 ***@***.***>
Sent: Tuesday, April 16, 2024 3:20 PM
To: MODFLOW-USGS/modflow6 ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [EXTERNAL] [MODFLOW-USGS/modflow6] mine pit infilling with tailings/rock (Discussion #1735)




 This email has been received from outside of DOI - Use caution before clicking on links, opening attachments, or responding.



I want to simulate mine pit infilling after mining completion. The mine pit will fill up with groundwater eventually, but will also be part of the process water system at the mine. For this reason, I simulated the mine pit during infilling using LAK so I can simulate the impacts of other parts of the water budget on the pit.

Now the mine is planning to also infill the pit lake with rock and/or tailings. Is appropriate to still use LAK for this?

The LAK covers many layers vertically, so I could use TVK to simulate the deposition in the LAK. I have never thought much about the k of the materials inside the LAK. Are cells inside a multilayer layer LAK "active" or they they removed from the model solution and replaced with the LAK cells?

Another thought I had was to try and modify the stage-volume-area curves to account the infilling (the added material will use up volume). However this would be time-varying as more material is added to the LAK through time and I don't think time-varying stage-volume is possible in LAK.

Or I could just not use LAK and use high k cells where the mine pit is open and use tvk to fill in the tailings/rock as it is deposited. However then I will not be able to account for the other pieces of the water balance easily (ET, precipitation, engineered pumping/discharge)

Any advice?

**Reply by RyanConway91** [reply]
Thanks for the reply. I like the idea of a series of separate simulations...

Another option to maybe get everything into one GWF model would be to create a unique lake for each stress period where each lake has connection data and stage-volume table based on the current pit fill status. I could activate and deactivate each lake as the pit fills up. Meanwhile TVK below the lake is accounting for tailings deposition/ongoing consolidation. 

Probably overkill... Im leaning towards dropping the LAK and just using TVK with high k cells for lake. I can use wells to account for the process water inputs and outputs.

**Comment by aprovost-usgs** [comment]
Ryan,

My colleague Chris Langevin gets credit for suggesting the series of simulations.  I like that approach, too.  I had considered defining a lake for each stress period, like you've suggested, but it wasn't clear to me how each lake could be made to pick up where the previous lake left off.  If you've thought of a way to pass the current water volume to the next lake, that would be a slick way to do the simulation all in one go.  There's a setting for STAGE in the PERIOD block, but its only for constant-stage lakes, so that wouldn't appear to be the answer. And it doesn't appear possible to use the mover (MVR) to transfer the water from one lake to the next.  My colleague Joe Hughes has suggested that if the lake is smaller than a grid cell, it might be possible to change the stage-area-volume table in laksettings.  All things considered, I agree with you that the series-of-simulations and high-k approaches are looking more promising.

-- Alden

Alden Provost
US Geological Survey
Integrated Modeling and Prediction Division
703-648-5883 (office)


________________________________
From: RyanConway91 ***@***.***>
Sent: Tuesday, April 23, 2024 6:48 PM
To: MODFLOW-USGS/modflow6 ***@***.***>
Cc: Provost, Alden ***@***.***>; Comment ***@***.***>
Subject: [EXTERNAL] Re: [MODFLOW-USGS/modflow6] mine pit infilling with tailings/rock (Discussion #1735)




 This email has been received from outside of DOI - Use caution before clicking on links, opening attachments, or responding.



Thanks for the reply. I like the idea of a series of separate simulations...

Another option to maybe get everything into one GWF model would be to create a unique lake for each stress period where each lake has connection data and stage-volume table based on the current pit fill status. I could activate and deactivate each lake as the pit fills up. Meanwhile TVK below the lake is accounting for tailings deposition/ongoing consolidation.

Probably overkill... Im leaning towards dropping the LAK and just using TVK with high k cells for lake. I can use wells to account for the process water inputs and outputs.
